source/ranges.tex

```diff
@@ -4514,7 +4521,7 @@
 \tcode{iterator_traits<iterator_t<\exposid{Base}>>::iterator_category}.
 \begin{itemize}
 \item
-If \tcode{is_lvalue_reference_v<invoke_result_t<F\&, range_reference_t<\exposid{Base}>>>}
+If \tcode{is_lvalue_reference_v<invoke_result_t<@\exposidnc{maybe-const}@<Const, F>\&, range_reference_t<\linebreak\exposid{Base}>>>}
```

Comment:
These `@` characters appear in the rendered PDF.

```suggestion
If \tcode{is_lvalue_reference_v<invoke_result_t<\exposidnc{maybe-const}<Const, F>\&, range_reference_t<\linebreak\exposid{Base}>>>}
```

Reply:
fixed